lib/PHPExif/Exif.php

```diff
@@ -1023,10 +1023,10 @@ public function getLatitude() : float|false
     /**
      * Sets the imgDirection value
      *
-     * @param string $value
+     * @param float $value
      * @return \PHPExif\Exif
      */
-    public function setImgDirection(string $value) : Exif
+    public function setImgDirection(float $value) : Exif
```

Comment:
Why is this a `float` only and not falsable (or even better nullable) like the other two attributes (latitude and longitude) above?